content/blog/do-not-miss-the-innovation-campus-at-discover-2024.md

```diff
@@ -0,0 +1,53 @@
+---
+title: do not miss The Innovation Campus at Discover 2024
```

Comment:
Title: Do not miss The Innovation Campus at HPE Discover 2024

Do with capital D

Shouldn't add "HPE" in front of Discover 2024 throughout the post?